package.json

```diff
@@ -43,11 +43,11 @@
     "prepublishOnly": "nr build",
     "release": "bumpp && npm publish",
     "start": "esno src/index.ts",
-    "test": "vitest",
+    "test": "FORCE_COLOR=3 vitest",
```

Comment:
The ENV variable `FORCE_COLOR=3` is required to enable color in Ansis, because `vitest` is not TTY.

Defaults behaviour:

- FORCE_COLOR=0       // 2 colors (no color)
- FORCE_COLOR=1       // 16 colors
- FORCE_COLOR=2       // 256 colors
- FORCE_COLOR=3       // truecolor